<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsageStatsOptionPage</class>
 <widget class="QWidget" name="UsageStatsOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_AllowPrivate">
     <property name="text">
      <string>Allow sending private data (IP, FC CPU numbers):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cb_AllowSending">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_AllowSending">
     <property name="text">
      <string>Allow sending application usage statistics to the project authors:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_UUID">
     <property name="text">
      <string>&lt;UUID&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cb_AllowPrivate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDebugLogLevel">
     <property name="text">
      <string>Debug logging level:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbDebugLogLevel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_AllowSending</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_AllowPrivate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_AllowSending</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_AllowPrivate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
